web/static/aws/connect/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>autokitteh AWS Integration</title>
+  <link rel="apple-touch-icon" sizes="180x180" href="/static/apple-touch-icon.png" />
+  <link rel="icon" type="image/png" sizes="32x32" href="/static/favicon-32x32.png" />
+  <link rel="icon" type="image/png" sizes="16x16" href="/static/favicon-16x16.png" />
+  <link rel="manifest" href="/static/site.webmanifest" />
+  <link rel="stylesheet" type="text/css" href="styles.css" />
+  <meta name="robots" content="noindex, nofollow" />
+</head>
+
+<body>
+  <a href="/" target="_self">
+    <img class="banner" src="/static/banner.svg" alt="autokitteh" />
+  </a>
+
+  <table class="title">
+    <td><img class="logo" src="/static/images/aws.svg" alt="Logo" /></td>
+    <td>
+      <h1>AWS - Create a New Connection</h1>
+    </td>
+  </table>
+
+
+  <form id="save-form" method="post" action="/aws/save">
+    <div class="form-group">
+      <label for="region"> Region </label>
+      <input type="text" name="region" autocomplete="off" spellcheck="false" class="text" required />
```

Comment:
Should this be a drop-down selection instead of free-form text?

Reply:
done, combined